app/registration/Address.tsx

```diff
@@ -0,0 +1,219 @@
+import { ErrorMessage } from '@hookform/error-message';
+import { Controller, useFormContext } from 'react-hook-form';
+import Select from 'react-select';
+import { countries, postalCodeFormats, Addresses } from './constants';
+import clsx from 'clsx';
+
+export default function Address() {
```

Comment:
You use different style of functions in your app. Chose one (arrow or function declaration)

Reply:
Changed function declaration for Address component into arrow funciton.